db/model/subject.js

```diff
@@ -344,8 +344,14 @@ module.exports = function subject(seq, dataTypes) {
 
         if (inst.getDataValue('isPublished')) {
           if (inst.previous('isPublished')) {
-            common.publishChange(inst, eventName.upd, changedKeys,
+            // Send event for updation of tags, first delete and then add
```

Comment:
Yes! Better!

One small thing in this comment line: "updation" is not a word. I think the comment should be:
```
/*
 * If tags were updated, send a "delete" event followed by an "add" event
 * so that perspectives using subject tag filters will get the right realtime
 * events. If subject tags were not updated, just send the usual "update"
 * event.
 */
```